clang-tools-extra/clang-tidy/modernize/UseRangesCheck.cpp

```diff
@@ -166,6 +166,15 @@ utils::UseRangesCheck::ReplacerMap UseRangesCheck::getReplacerMap() const {
   return Result;
 }
 
+UseRangesCheck::UseRangesCheck(StringRef Name, ClangTidyContext *Context)
+    : utils::UseRangesCheck(Name, Context),
+      UseReversePipe(Options.get("UseReversePipe", false)) {}
```

Comment:
this option could be moved to utils::UseRangesCheck to avoid duplications

Reply:
The issue with that is a downstream user who wants to implement this check for their library may not be able to support the alternative syntax.
The other way to get this behaviour would be a runtime parameter on the base class constructor to enable/disable the option. But that didn't sit right with me.

Reply:
ok